API/ev3_bluetooth.c

```diff
@@ -282,11 +282,9 @@ static int bindServerSocket () {
 }
 
 static BluetoothConnectionHandle acceptIncomingConnection () {
-    struct sockaddr_rc remoteAddress = { 0 };
-    socklen_t addressLength = sizeof(remoteAddress);
     BluetoothConnectionHandle c;
     do {
-        c = accept(serverSocket, (struct sockaddr *) &remoteAddress, &addressLength);
```

Comment:
What missing initialization do you mean? I don't see how this would triggger a warning, but passing NULL should be ok, according to man page.

Reply:
To fix [this](https://travis-ci.org/simonedegiacomi/EV3-API/builds/648101867#L2020) warning

Reply:
Ah, this should happen with GCC >=5, I think. [Source.](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=36750). But ye, change looks ok.